wiki/Ways-Of-Working.md

```diff
@@ -1,21 +1,104 @@
+
+Occasionally, quality checks will erroneously flag code which is used throughout our codebase, such as `useEffect`. These can be ignored.
+
+# Standup
+
+1. Standups are currently M, T, & Th at 12:00 EST for 15 minutes, followed by an optional "parking lot" if there are further items that need discussing. 
+2. Please make sure all your stories are up-to-date before standup, and have been assigned the correct status. 
+    - Remember to associate any PRs you submit with the ticket it closes (and automation will do the rest).
+3. When you give your standup, try to stay concise, providing a general status update for various tickets, and briefly flagging any complications.
+    - Tickets and PRs should be referenced by both their assigned numbers (e.g. "#4572") and their descriptive titles.
+    - All complications can be handled in parking lot (or "p-lot"). We bracket them in the main standup session to keep the call short and sweet for engineers.
+4. The purpose of standup is to keep your fellow team members (both engineers and leads) informed. Standups should be given in this spirit, addressing the team broadly, and ensuring that enough contextual information is provided that your colleagues can follow along.
+5. Please keep your camera on whenever possible, and ensure you are in a relatively quiet space with clear audio quality.
+
+# Deployment and QA schedule
+
+Agile Release Trains (aka [ART, from SAFe5](https://v5.scaledagileframework.com/agile-release-train/)) run 4.5 days a week, Monday morning through mid-day Friday. We do not deploy on Fridays after 13:00 EST. If QA has been approved by 13:00, we will ship the release; otherwise we don't ship until Monday.
+
+Over the course of the day, pull requests are merged into the Master branch on an ad hoc basis. On merge, the code is automatically pushed to our Beta (QA) server. Each morning, before the next round of PR merges, Beta ships a new release to the live site. Typically, a working session (aka "PR Party") is held to make sure everything shippable will be included in the next day's release.
+
+Currently, Product team is responsible for QA'ing the day's release on Beta before the following morning. As of 230831, we have hired a QA Engineer who will begin taking on this work as well as executing the automation plan; expect this part of process to be in flux.
 
-## Github Quality Checks
+Engineering is responsible for informing Product that release on Beta is ready for QA. Effectively, this means everything that is ready for QA that day has been deployed to Beta. 
 
-Pull requests to master _should_ pass Github's code quality checks. (Nb: Not the test suite run by our CI, but those quality checks identified in the PR's files/diff.) PR authors _should_ resolve these checks themselves.
+Our Beta/QA server can be found at `qa.commonwealth.im`.
```

Comment:
Note that custom domains are available at osmosis.qa.commonwealth.im and dydx.qa.commonwealth.im